content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -186,6 +184,127 @@ Resource associated with Activity:
 This output has both the span created to track work in the route, and an
 automatically-created span that tracks the inbound ASP.NET Core request itself.
 
+## Send traces to a collector
+
+The [OpenTelemetry Collector](/docs/collector/getting-started/) is a vital
+component of most production deployments. A collector is most beneficial in the
+following situations, among others:
+
+* A single telemetry sink shared by multiple services, to reduce overhead of
+  switching exporters
+* Aggregate traces across multiple services, running on multiple hosts
+* A central place to process traces prior to exporting them to a backend
+
+### Configure and run a local collector
+
+First, save the following collector configuration code to a file under `/tmp/`:
+
+```yaml
+# /tmp/otel-collector-config.yaml
+receivers:
+  otlp:
+  protocols:
+    grpc:
```

Comment:
nit: adding `grpc` here is not requered since the changes in the c# file use `http` only. I tend to prefer the minimal configuration required.